src/modules/users/components/user-form.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { reduxForm, FieldArray } from "redux-form";
+import { FormField } from "@salsita/react-forms";
+import { FormFieldSelect } from "@salsita/react-forms";
+import { FormValidations } from "@salsita/react-forms";
+import { connect } from "react-redux";
+
+import { getSkills } from "modules/entities/entities-selectors";
+
+const validateFirstName = FormValidations.notEmptyString(
+  "first name cannot be empty"
+);
+const validateLastName = FormValidations.notEmptyString(
+  "last name cannot be empty"
+);
+
+const validateSkills = skills => {
+  if (!Boolean(skills)) {
```

Comment:
Not saying this is a problem. Someone likes `Boolean` and I don't have a problem with it. So just FYI, this line is the same as
```js
if (!skills) {
```

I would use the shorter version btw.

Reply:
Yep the shorter version looks better